README.md

```diff
@@ -5,7 +5,7 @@ Modeling of published clinical Cimetidine-Metformin-DDI studies for model evalua
 Within this repository, we share a PK-Sim project file containing simulations of all published clinical studies used to evaluate the predictive performance of our models regarding the Cimetidine-Metformin-DDI, including the respective observed data. For further details and documentation please refer to [[1](#reference)].
```

Comment:
Please change "share PK-Sim project" to "Share PK-Sim snapshot" (or how is this formulated in other DDI repos...)